.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+	parser: '@typescript-eslint/parser',
```

Comment:
Deze word normaal automatisch gezet met 1 van de onderstaande extends of had je ergens issues?

Reply:
Ik kreeg volgende error: 
![image](https://user-images.githubusercontent.com/85545258/146179257-35bb8d4a-3d1a-43b7-8839-82037beb1cc8.png)